test/e2e/common/node/security_context.go

```diff
@@ -633,6 +639,249 @@ var _ = SIGDescribe("Security Context", func() {
 			}
 		})
 	})
+
+	f.Context("SupplementalGroupsPolicy [LinuxOnly]", framework.WithFeatureGate(features.SupplementalGroupsPolicy), func() {
```

Comment:
This currently still has to have `framework.Feature`, to support jobs which don't skip with --label-filter.

As it stands right now, the test will run in https://testgrid.k8s.io/sig-release-master-blocking#conformance-ga-only (filters only by the traditional `\[Feature:.*\]` regexp) and fail there because the feature gate will be set to false.

Reply:
Oh, thanks for pointing out. Then I will add `feature.SupplementalGroupsPolicy` again. 🙇

Reply:
Last week we merged https://github.com/kubernetes/kubernetes/pull/130655 which changes this: now `[Feature:OffByDefault]` gets added by `WithFeatureGate`, which mean you no longer need a separate `WithFeature`.

Still need to communicate our revised plan more widely...

Reply:
Also https://github.com/kubernetes/kubernetes/pull/130908

Reply:
But you don't need to do this before merging. It can be cleaned up later.

Reply:
Cool. Thanks for the info! I will do it later.

Reply:
This requires a feature label for now given:
> This feature depends on CRI runtime support. Thus, this PR performs optional E2E tests on both CRI runtimes that supports/does not support the feature.

Reply:
continued in this thread:  https://github.com/kubernetes/kubernetes/pull/130210#discussion_r2007715601